spec/amp-actions-and-events.md

```diff
@@ -195,17 +195,32 @@ For example, the following is possible in AMP.
     <th>Description</th>
   </tr>
   <tr>
-    <td>hide</td>
+    <td>`hide`</td>
     <td>Hides the target element.</td>
   </tr>
   <tr>
-    <td>show</td>
+    <td>`show`</td>
     <td>Shows the target element.</td>
   </tr>
   <tr>
-    <td>toggleVisibility</td>
+    <td>`toggleVisibility`</td>
     <td>Toggles the visibility of the target element.</td>
   </tr>
+  <tr>
+    <td>`scrollTo(duration=INTEGER, position=STRING)`</td>
+    <td>Scrolls the element into view with a smooth animation. If defined,
+    `duration` specifies the length of the animation in milliseconds. `position`
+    is optional and takes one of `top`, `center` and `bottom` defining where
+    in the viewport the element will be at the end of the scroll.</td>
+  </tr>
+  <tr>
+    <td>`focus`</td>
+    <td>Makes the target element gain focus.</td>
+  </tr>
+  <tr>
+    <td>`blur`</td>
```

Comment:
`blur` is not good to expose because it breaks A11Y. It moves the focus back to `body` which is almost never a desired behaviour. If author wants something to lose focus, they need to decide what makes sense to get the focus and use `focus` action instead. 

Was there a particular use-case you had in mind for this?